api/base/serializers.py

```diff
@@ -82,15 +81,14 @@ def to_representation(self, value):
         """
         url = super(JSONAPIHyperlinkedIdentityField, self).to_representation(value)
 
-        if self.meta:
-            meta = {}
-            for key in self.meta:
-                meta[key] = _rapply(self.meta[key], _url_val, obj=value, serializer=self.parent)
-            self.meta = meta
-
-            return {'links': {self.link_type: {'href': url, 'meta': self.meta}}}
-        else:
-            return {'links': {self.link_type: url}}
+        meta = {}
+        for key in self.meta or {}:
+            if key == 'count':
+                show_related_counts = self.context['request'].query_params.get('related_counts', False)
```

Comment:
This is probably farther along the process than it should be for this question, but: would it make sense to make this a bit more generic and also a bit more granular? If the query parameter were `relationship_meta[children]=true` then we could get the child count without also getting the contributor count. If both are particularly large, and we only needed one, this could save some trouble. Or if something else in a meta field were similarly expensive. If we wanted to make it granular but didn't care about other meta items, `relationship_counts[contributors]=true` might be a good alternative.